runners-msi/common/common-directories.wxi

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
That's checked, they're all good. :+1: 

What I actually did was deleted the runners project, and adjusted NUnit-Everything to essentially be what runners used to be. But the above made a clearer explanation!